docs/topics/Gateway.md

```diff
@@ -677,7 +677,13 @@ A user's presence is their current state on a guild. This event is sent when a u
 |-------|------|-------------|---------|
 | name | string | the game's name | Always |
 | type | integer | see [Game Types](#DOCS_GATEWAY/game-types)  | Always |
-| url | ?string | stream url, is validated when type is 1  | When type is 1 |
+| url? | ?string | stream url, is validated when type is 1  | When type is 1 |
+| timestamps? | see [timestamps](#DOCS_GATEWAY/game-timestamps) | Unix timestamps for start and/or end of the game | Sometimes |
+| application_id? | snowflake | Application ID for the game | Sometimes |
+| details? | ?string | What the player is currently doing | Sometimes |
+| state? | ?state | The user's current party status | Sometimes |
+| party? | see [party](#DOCS_GATEWAY/game-party) | Information for the current party of the player | Sometimes |
+| assets? | see [assets](#DOCS_GATEWAY/game-assets) | Images for the presence and their hover texts | Sometimes |
```

Comment:
Do bots get the secrets from presence? I assume no?

Reply:
I'm unsure, didn't check that specifically